core/ast_node.js

```diff
@@ -257,16 +256,18 @@ Blockly.ASTNode.prototype.findNextEditableField_ = function(location,
 /**
  * Get either the previous editable field, or get the first editable field for
  * the given input.
- * @param {!Blockly.Field} location The current location of the cursor.
+ * @param {!Blockly.Field|Blockly.Connection} location The current location of
+ *     the cursor.
```

Comment:
"The current location of the cursor, which must be a field or connection."

Reply:
fixed